components/ActionMenus/ItemActionMenu.tsx

```diff
@@ -12,16 +12,20 @@ import { Item } from '@/utils/types';
 
 type ItemActionMenuProps = {
   item: Item;
+  variant?: 'outline' | 'solid' | 'ghost';
 };
 
-export const ItemActionMenu: React.FC<ItemActionMenuProps> = ({ item }) => {
+export const ItemActionMenu: React.FC<ItemActionMenuProps> = ({
+  item,
+  variant = 'outline',
+}) => {
```

Comment:
Wonder if we should add the equip action to this, since it's impossible to equip some items in the character card now, if there are more 2 that are claimed

Reply:
Added in f2b29eb9aeee29efbe30bf0dd5ea82ce0479c919